src/pseudopeople/configuration/interface.py

```diff
@@ -16,6 +16,12 @@
     $ user_config = {"decennial_census": {"row_noise": {"probability": {"omit_rows": 0.1},},},}
     $ psp.get_config("decennial_census", user_config)
 
+Note that when specifying a value to override in the configuration, users must specify the specific node they wish to
+change. Configuration is a hierarchical structure and to must properly source the lower levels. The configuration
+levels include dataset, column or row noise, noise type, and probability. Not sourcing values in a user provided
+configuration correctly will raise a ConfigurationKeyError if the lookup fails. If an invalid value is provided, such as
+providing 1.5 for a probability, a ConfigurationError will be raised.
+
```

Comment:
I tried adding this to the actual docstring below but that didn't seem to work.  The `get_config `function only raises a value error but the function it is wrapping can raise these two errors so I wanted that to be documented somehere.

Reply:
Can you add a raises section to the docstring for `get_configuration` and see if that solves this issue?

Reply:
See my PR. We have to nitpick the ConfigurationError